doc/developers/testsuite.html

```diff
@@ -546,6 +522,43 @@
       files.
     </p>
 
+
+    <a name="variants"></a>
+    <h3>Tests with multiple comparison files</h3>
+    <p>
+      Sometimes it is necessary to provide multiple comparison files for a
+      single test. Additional comparison files have the same path as the
```

Comment:
-> [single test], for example because you want to test code on multiple platforms that produce different output files that, nonetheless, all should be considered correct. An example would be tests that use the `rand()` function that is implemented differently on different platforms.